grpcService/cmd/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"net"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"google.golang.org/grpc/reflection"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"google.golang.org/grpc"
+
+	pb "github.com/timoteoBone/microservice-project/grpcService/pkg/pb"
+	"github.com/timoteoBone/microservice-project/grpcService/pkg/user"
+)
+
+func main() {
+
+	var db *sql.DB
+
+	db, err := sql.Open("mysql", "root:PewDiePie8!!@tcp(127.0.0.1:3306)/test?parseTime=true")
```

Comment:
here use vars to edit this string. 
```suggestion
	db, err := sql.Open("mysql", fmt.Sprintf("root:%s@tcp(%s:%s/%s?parseTime=True",cfg.usr,cfg.pswr,cfg.Hots,cfg.Port,cfg.DefaultDB)
```